homeassistant/components/mobile_app/webhook.py

```diff
@@ -295,10 +298,11 @@ async def webhook_render_template(hass, config_entry, data):
     resp = {}
     for key, item in data.items():
         try:
-            tpl = item[ATTR_TEMPLATE]
-            attach(hass, tpl)
+            tpl = template.Template(item[ATTR_TEMPLATE])
+            tpl.ensure_valid()
+            tpl.hass = hass
             resp[key] = tpl.async_render(item.get(ATTR_TEMPLATE_VARIABLES))
-        except TemplateError as ex:
+        except Exception as ex:  # pylint: disable=broad-except
```

Comment:
```suggestion
        except TemplateError as ex:
```

Should be good to keep it as original after https://github.com/home-assistant/core/pull/41719